### PR TITLE
modify some config and resolve a bug for genrs

### PR DESCRIPTION
**Pull request by aressu1985**
modify some config and resolve a bug for genrs

### CODE REVIEW

File: src/main/java/io/mo/Tester.java [hunk 1]
```diff
@@ -165,8 +165,10 @@ public static void main(String[] args){
 
     public static void run(File file){
         if(file.isFile()){
-            if(file.getName().endsWith(".result"))
+            if(!(file.getName().endsWith(".sql") || file.getName().endsWith(".sql"))) {
```
Comment (sukki37) [inline src/main/java/io/mo/Tester.java:168]: why did u repeat the same condition?

File: src/main/java/io/mo/Tester.java [hunk 2]
```diff
@@ -186,8 +188,10 @@ public static void run(File file){
 
     public static void generateRs(File file){
         if(file.isFile()){
-            if(file.getName().endsWith(".result"))
+            if(!(file.getName().endsWith(".sql") || file.getName().endsWith(".sql"))) {
```
Comment (sukki37) [inline src/main/java/io/mo/Tester.java:191]: why did u repeat the same condition twice?

File: src/main/java/io/mo/Tester.java [hunk 3]
```diff
@@ -209,8 +213,10 @@ public static void generateRs(File file){
 
     public static void debug(File file){
         if(file.isFile()){
-            if(file.getName().endsWith(".result"))
+            if(!(file.getName().endsWith(".sql") || file.getName().endsWith(".sql"))) {
```
Comment (sukki37) [inline src/main/java/io/mo/Tester.java:216]: same question:https://github.com/matrixorigin/mo-tester/pull/105/files#r969215071

File: src/main/java/io/mo/Tester.java (hunk 1 above)
Comment (aressu1985) [reply src/main/java/io/mo/Tester.java:168]: I made a mistake

**Pull request closed without merging**